055-ChaosStudio4AKS/Coach/README.md

```diff
@@ -0,0 +1,74 @@
+# What The Hack - ChaosStudio4AKS - Coach Guide
+
+## Introduction
+
+Welcome to the coach's guide for the ChaosStudio4AKS What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+This hack includes an optional [lecture presentation](Lectures.pptx) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+
+- Challenge 00: **[Prerequisites - Ready, Set, GO!](./Solution-00.md)**
+	 - Prepare your workstation to work with Azure.
+- Challenge 01: **[Is your Application ready for the Super Bowl](./Solution-01.md)**
+	 - How does your application handle failure during large scale events?
+- Challenge 02: **[My AZ burned down, now what?](./Solution-02.md)**
```

Comment:
Names of the solutions should match.  Please make sure to align with challenge titles as well.

Reply:
I think I got them all, as they are little ones and my eyes are tired of this hack, I may have missed some